CMSgov/mint-app
# [EASI-4380] Shared systems updates

# EASI-4380


## Changes and Description

- Fixed incorrect usage of `Trans` component in help center
- Fixed bugginess of modal when url is pasted in browser
- Default to query param `page=1` if no page param present
- Updated Shared Systems to match Figma



## How to test this change

Verify changes match figma
Verify modal closes properly is url is pasted in browser


## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
